.buildkite/scripts/steps/gce-cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+export ACCOUNT_KEY_SECRET=$(vault kv get -field=client_email $VAULT_PATH)
```

Comment:
For reading information from the vault, we kind of followed a pattern to put those into the pre-command hook.

Reply:
Thanks, good point

Reply:
Now the variables are being unset regardless of command exit code. What do you think if I keep it within the step script? For me it looks quite straightforward and doesn't require any additional if conditions

Reply:
The intention is to have the script executable from ci as well as locally for debug. This is why it's better to set up the creds as part of the hook in ci.